org.abapgit.adt.ui/src/org/abapgit/adt/ui/internal/wizards/AbapGitWizardPageObjectsSelectionForPull.java

```diff
@@ -334,51 +333,64 @@ private IRepositoryModifiedObjects findParent(IOverwriteObject obj) {
 
 	private void tryLazyLoad() {
 		var tree = this.modifiedObjTreeViewer.getTree();
-		ScrollBar vBar = tree.getVerticalBar();
-		if (vBar == null || vBar.getMaximum() == 0) {
+		ScrollBar verticalScrollBar = tree.getVerticalBar();
+		if (verticalScrollBar == null || verticalScrollBar.getMaximum() == 0) {
 			return;
 		}
 
-		int scrollPosition = vBar.getSelection() + vBar.getThumb();
-		// Trigger loading when scrolled to 90% of the maximum for smoother experience
-		int threshold = (int) (vBar.getMaximum() * 0.9);
+		int scrollPosition = verticalScrollBar.getSelection() + verticalScrollBar.getThumb();
+		int threshold = (int) (verticalScrollBar.getMaximum() * LAZY_LOAD_THRESHOLD);
```

Comment:
Would adding _Scroll_ to the constant make sense ?
Like SCROLL_LAZY_LOAD_THRESHOLD or SCROLL_THRESHOLD_LAZY_LOAD.